common.blocks/button/button.bemhtml

```diff
@@ -1,4 +1,11 @@
 block('button')(
+    def()(function() {
+        var isRealButton = (['button', 'input'].indexOf(apply('tag')) >= 0) &&
```

Comment:
Ok, now I get it. 

But at least, please, let's change this ridiculous `['button'].indexOf(tag) >= 0` to `tag === 'button'`